File: SW Expert Academy/D1/2070 큰 놈, 작은 놈, 같은 놈.cpp
/*******************************************
 * SW Expert Academy 2070: Å« ³ð, ÀÛÀº ³ð, °°Àº ³ð
 * D1
 * 21.1.30 try: 1
 *******************************************/
#include <iostream>

int main(int argc, char** argv) {
	std::ios_base::sync_with_stdio(false);
	std::cin.tie(nullptr);
	std::cout.tie(nullptr);

	int test_case;
	int T;
	
	//freopen("input.txt", "r", stdin); //#include <cstdio>
	std::cin >> T;

	for (test_case = 1; test_case <= T; ++test_case) {
		int a, b;
		std::cin >> a >> b;

		char ans;
		if (a > b) ans = '>';
		else if (a < b) ans = '<';
		else ans = '=';
		
		printf("#%d %c\n", test_case, ans);
		//std::cout << "#" << test_case << ' ' << sum / 10 << '\n';
	}
	
	return 0;
}

File: SW Expert Academy/D1/2070 큰 놈, 작은 놈, 같은 놈.py
'''
 # SW Expert Academy 2070: 큰 놈, 작은 놈, 같은 놈
 # D1
 # 21.1.30 try: 1
'''
#sys.stdin = open("input.txt", "r") #import sys

T = int(input())
for test_case in range(1, T + 1):
    l = [int(x) for x in input().split()]

    if l[0] > l[1]:
        ans = '>'
    elif l[0] < l[1]:
        ans = '<'
    else:
        ans = '='
    # print("#{0} {1:.0f}".format(test_case, sum(l) / 10))
    # print("#%d %.0f" % (test_case, sum(l) / 10))
    print(f"#{test_case} {ans}")